translations/frontend/en.json

```diff
@@ -2259,7 +2259,7 @@
                             "without_device": "Entities without device"
                         },
                         "icon": "Icon",
-                        "introduction": "Use scenes to bring your home to live.",
+                        "introduction": "Use scenes to bring your home to life.",
```

Comment:
Translation files should not be changed via PRs. They are updated via the "lokalise" translation project.

Reply:
Oh thanks for the heds up, i wasn't aware of the workflow related to translation files. i removed the change from that file